Jenkinsfile

```diff
@@ -100,6 +100,23 @@ pipeline {
                 }
             }
         }
+        stage('Run OWASP ZAP security framework') {
+            agent {
+                docker {
+                    image 'owasp/zap2docker-stable'
+                    args '-u root:sudo --net demo -v ${WORKSPACE}:/zap/wrk:rw'
+                    reuseNode true
+                }
+            }
+            environment {
+                CONTAINER_HTTP_URL = "http://${APP_NAME}:8080"
+            }
+            steps {
+                script { STAGE = env.STAGE_NAME }
+                sh "/zap/zap-baseline.py -d -a -j -t http://${DEV_IP}/${APP_NAME}/ -m 1 -c owasp_zap.conf -r testreport.html -w testreport.md -x testreport.xml -n owasp_zap.context"
```

Comment:
@jknight-liatrio - Can we specify the name of the report to be something like OWASPtestreport.html or zaptestreport.html.  Thoughts?